agent/build.gradle

```diff
@@ -54,12 +54,12 @@ shadowJar {
 	// Needed as a workaround for https://github.com/johnrengelman/shadow/issues/521
 	inputs.property("debug", project.hasProperty("debug"))
 
-	archiveBaseName = 'teamscale-jacoco-agent'
+	baseName = 'teamscale-jacoco-agent'
 	classifier = null
 	// since this is used as an agent, we want it to always have the same name
 	// otherwise people have to adjust their -javaagent parameters after every
 	// update
-	archiveVersion = null
+	version = null
```

Comment:
I think we should move that version number reset out of the shadowJar block as it suggests that this is a property of the jar task, but actually this sets the version of the complete project.